File: src/main/java/org/intellij/tool/utils/TimeUtils.java
package org.intellij.tool.utils;

import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;

public class TimeUtils {

    static String defaultPattern = "yyyy-MM-dd HH:mm:ss.SSS";

    public static String getCurrentTime(String pattern) {
        if (StringUtils.isBlank(pattern)) {
            pattern = defaultPattern;
        }
        return LocalDateTime.now().format(DateTimeFormatter.ofPattern(pattern));
    }
}

File: src/main/java/org/intellij/tool/utils/TimeUtils.kt
package org.intellij.tool.utils

import java.time.LocalDateTime
import java.time.format.DateTimeFormatter

object TimeUtils {
    private var defaultPattern: String = "yyyy-MM-dd HH:mm:ss.SSS"

    fun getCurrentTime(pattern: String?): String {
        return LocalDateTime.now().format(DateTimeFormatter.ofPattern(pattern ?: defaultPattern))
    }
}
